server.js

```diff
@@ -3,9 +3,50 @@ require('habitat').load();
 var express = require('express');
 var app = express();
 var routes = require('./routes');
+var compression = require('compression');
+var helmet = require('helmet');
+var frameguard = helmet.frameguard;
```

Comment:
small nit - we might as well do what we do elsewhere and call helmet.frameguard instead of referencing it in a local variable.